### DISCUSSION TITLE
Projeto de gerenciamento

### DISCUSSION THREAD

**Discussion by romulociro** [frontendbr/forum#2330] [Dicas]
Boa tarde,

Preciso criar um projeto para gerenciar um sistema de cultivo.

Hoje o cliente usa uma planilha excel e ele quer ter no celular dele um app para acompanhar.

Ele quer que o funcionário alimente o DB pelo App, quando plantou, dividido por área para saber qual o dia vai colher, adubar etc...

Alguém pode me dizer um stack legal para desenvolver esse App?

Pensei em fazer um dashboard com react e usar react native pro app além de um banco de dados MySQL.

Desde já agradeço pelas dicas.

**Comment by felquis** [comment]
As ferramentas que você escolheu estão ótimas Romulo.

**Comment by ghost** [comment]
Oi romulociro.

> Boa tarde,
> Preciso criar um projeto para gerenciar um sistema de cultivo.
> Hoje o cliente usa uma planilha excel e ele quer ter no celular dele um app para acompanhar.
> Ele quer que o funcionário alimente o DB pelo App, quando plantou, dividido por área para saber qual o dia vai colher, adubar etc...
> Alguém pode me dizer um stack legal para desenvolver esse App?
> Pensei em fazer um dashboard com react e usar react native pro app além de um banco de dados MySQL.
> Desde já agradeço pelas dicas.

você já pensou em criar um protótipo de tela no figma ou utilizar a plataforma expo para prototipar o app? - essas são duas ideias interessantes inicialmente para elaborar o app.
